src/common/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+Button.propTypes = {
+  onClick: PropTypes.func,
+  children: PropTypes.string.isRequired,
+}
+
+export default function Button({ children, onClick }) {
+  return <BlockButton onClick={onClick}>{children}</BlockButton>
```

Comment:
The name BlockButton might sound a little misleading. I know it references the display: block in your styling, but someone might assume this is a Button with a block function. Maybe ButtonStyled would be an easy alternative?